libcerf/tests/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
+
+set -euo pipefail
+
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
+
+echo "This package only provides libraries and headers and currently has no post-build tests"
```

Comment:
Should we still do the package install? That feels like it would be testing very little about the plan itself, but it's at least something.

Reply:
I don't think there would be any value in that. You'd essentially be (IMO) asserting that plan-build did its thing and the hab cli functions.